README.md

```diff
@@ -0,0 +1,55 @@
+# in-toto Layout Creation Wizard
```

Comment:
Consider saying somewhere in the README that an instance of the layout tool is available and/or link to it.

Reply:
Done